### PR TITLE
ARM64 cleanup

### PR DESCRIPTION
**Pull request by carlocaione**
Simple patchset made by purely non-functional changes to fix comments, names, indentation, etc...

### CODE REVIEW

**Comment by carlocaione** [comment]
`buildkite` failure unrelated

File: arch/arm/core/aarch64/reset.S [hunk 1]
```diff
@@ -22,34 +21,27 @@ _ASM_FILE_PROLOGUE
...
- *
- * @return N/A
  */
+
 GTEXT(__reset)
 SECTION_SUBSEC_FUNC(TEXT,_reset_section,__reset)
 
-/*
- * The entry point is located at the __reset symbol, which is fetched by a XIP
- * image playing the role of a bootloader, which jumps to it, not through the
```
Comment (andrewboie) [inline arch/arm/core/aarch64/reset.S:49]: are this and other multi-line comments that were removed in this patch no longer correct?
when it comes to assembly files I often think it's worth being verbose when explaining things.

File: include/arch/arm/aarch64/cpu.h [hunk 2]
```diff
@@ -28,11 +28,11 @@
 
 #define SCTLR_EL1_RES1		(BIT(29) | BIT(28) | BIT(23) | \
 				BIT(22) | BIT(20) | BIT(11))
-#define SCTLR_M_BIT		BIT(0)
-#define SCTLR_A_BIT		BIT(1)
```
Comment (andrewboie) [inline include/arch/arm/aarch64/cpu.h:32]: does this header get pulled in by public, application-facing headers like kernel.h or is it just used by the arch/ code? maybe we could move it to arch/arm/include area, the definitions here aren't namespaced in any particular way

**Review by tyrantlizard77 — COMMENTED** [review]
generally looks good just some questions

File: arch/arm/core/aarch64/reset.S (hunk 1 above)
Comment (carlocaione) [reply arch/arm/core/aarch64/reset.S:49]: That was a leftover from the old Cortex-M file that I used as reference at the beginning of the port. XIP image is not currently supported and I'm not even sure that it works on AArch64 the same way it does on the ARMv7. So I just decided to take that out.

File: include/arch/arm/aarch64/cpu.h (hunk 2 above)
Comment (carlocaione) [reply include/arch/arm/aarch64/cpu.h:32]: There is a bit of everything in this file including stuff used by drivers so I think we really need this in some accessible location in the include directory.

**Review by tyrantlizard77 — APPROVED** [review]

**Comment by carlocaione** [comment]
@Abhishek-brcm @sandeepbrcm are you ok with this?

**Comment by carlocaione** [comment]
@andrewboie @nashif can we please merge this?

**Review by ioannisg — APPROVED** [review]
the cleanup looks fine to me.

**Comment by ioannisg** [comment]
> Simple patchset made by purely non-functional changes to fix comments, names, indentation, etc...

btw there still exist some minor functional changes, e.g. chaning ASM instructions ...

**Review by sandeepbrcm — APPROVED** [review]

**Pull request merged**